src/Sarif/Baseline/ResultMatching/SarifLogMatcher.cs

```diff
@@ -116,10 +116,10 @@ private SarifLog BaselineSarifLogs(IEnumerable<Run> previous, IEnumerable<Run> c
         {
             // Spin out SARIF logs into MatchingResult objects.
             List<ExtractedResult> baselineResults = 
-                previous == null ? new List<ExtractedResult>() : GetMatchingResultsFromRuns(previous);
```

Comment:
>GetMatchingResultsFromRuns [](start = 65, length = 26)

I didn't like this name because we don't know whether the results we extract will match. Another possible name for `ExtractedResult` is `MatchableResult`, and then this method could be called `GetMatchableResultsFromRuns`.